Makefile

```diff
@@ -381,7 +381,12 @@ istioctl-install:
 .PHONY: test localTestEnv test-bins
 
 # Run coverage tests
-test: pilot-test mixer-test security-test broker-test galley-test common-test
+JUNIT_UNIT_TEST_XML ?= $(ISTIO_OUT)/junit_unit_tests.xml
+test:
```

Comment:
"test" is part of the default rule, so I'm afraid that this will break for devs that don't happen to already have "go-junit-report" installed.  It seems like there might need to be a rule/dependency install this if necessary, or instead not set a default and this make it opt-in behavior (by setting JUNIT_UNIT_TEST_XML), where the tee part can probably be made conditional using something like:
<pre>$(if $(JUNIT_UNIT_TEST_XML),|& tee >(go-junit-report > $(JUNIT_UNIT_TEST_XML)) )</pre>
or I suppose another option is to have a targets for the opt-in of junit behavior.

Reply:
I added a new target that check if go-junit-report is installed and install it if not.